src/raster/r.survey/r.survey.html

```diff
@@ -0,0 +1,290 @@
+Solid Angle of a surface is, by definition, equal to the projected spherical surface in the evolving sphere divided by the square of the radius of the sphere. As an aproximation, we use the ellipse surface area in place of the projected spherical surface in the evolving sphere. The maxSolidAngle map shows the value of the maximum Solid Angle among those measured from the different viewpoints. In this map, the closest pixels to the observation points have to be interpreted with special attention, considering that the error, respect to the real Solid Angle, can reach until the 10% in the immediate neighbor pixels
+<p>
+Other three maps (pointOfViewWithMin3dDistance, pointOfViewWithMmaxAngle and pointOfViewWithMmaxSolidAngle) are used to register, in each cell, the identifier of the viewpoints from where an observer can get, respectively, (i) the minimum values of 3D Distance, (ii) the maximum values of View Angle and (iii) the maximum value of the Solid Angle. Another relevant output is the numberOfViews map which portraits the number of viewpoints from where each pixel is visible.
+
+<h2>INSTALLATION</h2>
+<h3>LINUX:</h3>
+
+1) Download the compressed folder of r.survey from <a href="https://doi.org/10.5281/zenodo.3993140">https://doi.org/10.5281/zenodo.3993140</a> and decompress it. 
+<p>
+2) Copy the file <em>r.survey.py</em> into the folder <em>/usr/lib/grass78/scripts/</em>
+<p> 
+3) Copy the file <em>r.survey.html</em> and the folder <em>r.survey_figures</em> into the directory <em>/usr/lib/grass78/docs/html/</em>
+<p> 
+4) Open GRASS and run the following commands in order to verify if it works: 
+<dd><div class="code"><pre>r.survey --h</pre></div></dd>
+<dd><div class="code"><pre>g.manual r.survey</pre></div></dd>
+
+<h3>WINDOWS:</h3>
+
+Depending on the type of GRASS installation carried out, the directory paths can slightly vary. 
+<p>
+1) Download the compressed folder of r.survey from <a href="https://doi.org/10.5281/zenodo.3993140">https://doi.org/10.5281/zenodo.3993140</a> and decompress it. 
+<p>
+2) Copy the file named <em>r.survey.bat</em> into the following directory:
+
+<dd><em>C:\Program Files\GRASS GIS 7.8\bin\</em>  (regular installation)</dd>
+<dd>or</dd>
+<dd><em>C:\OSGeo4W64\apps\grass\grass78\bin\</em> (installation from OSGeo)</dd>
+<p>
+3) Copy the file <em>r.survey.py</em> into the followin folder:
+
+<dd><em>C:\Program Files\GRASS GIS 7.8\scripts\</em>  (regular installation)</dd>
+<dd>or</dd>
+<dd><em>C:\OSGeo4W64\apps\grass\grass78\scripts\</em> (installation from OSGeo)</dd>
+<p>
+4) Copy the file <em>r.survey.html</em> and the folder <em>r.survey_figures</em> into the following directory:
+
+<dd><em>C:\Program Files\GRASS GIS 7.8\docs\html\</em>  (regular installation)</dd>
+<dd>or</dd>
+<dd><em>C:\OSGeo4W64\apps\grass\grass78\docs\html\</em> (installation from OSGeo)</dd>
+<p>
```

Comment:
Please use `g.extention`- and it is also outdated (this is the GRASS GIS 8 branch of addons).

Reply:
Fine.. I'll do it